api/src/main/java/com/velocitypowered/api/proxy/Player.java

```diff
@@ -275,4 +275,11 @@ default void sendMessage(net.kyori.text.Component component) {
    */
   @Override
   boolean sendPluginMessage(ChannelIdentifier identifier, byte[] data);
+
+  /**
+   * Gets the player's client brand.
+   *
+   * @return the player's client brand
+   */
+  Optional<String> getClientBrand();
```

Comment:
This should be `@Nullable String` - going forward, we are eschewing all use of optionals, and they've already been removed in Polymer.